Role improved

basilgood opened the pull request.

basilgood committed:
Some code updates

basilgood committed:
New path to unix.socket

basilgood committed:
Code yaml corrected

basilgood committed:
Add item.url

basilgood committed:
Install lxd only from snap

basilgood committed:
Inventory update

basilgood committed:
Disable security apparmor for nested containers

basilgood committed:
Spell correct

megheaiulian commented on defaults/main.yml:
```diff
@@ -1,22 +1,21 @@
-lxd_apt_stable_repo: yes
-lxd_packages:
-  - lxd
-  - zfsutils-linux
```
Keep  `zfsutils-linux`.

basilgood committed:
Add dependencies

basilgood committed:
Inventory updates

Inventory localhost

No zfs for trusy

Default pool driver to dir

Default driver conditional

Pre task

Correct code

State absent

Add user travis

Create lxd group and add user to it

Xenial on travis

Install zfsutils

zfsutils-linux

List var/snap directory

Ls var

lxd init --auto

Lxd configuration only

Install lxd in travis

Install with sudo

Correct conditional

Restart lxd service

Install lxd from role

Restart services

Start lxd service

Masked no

Restart snap.lxd.daemon

Snap.lxc restarted

Install lxd only if not available

Lxd conditional
